### PR TITLE
19: Add Feeder motor to Shooter Subsystem

### LINKED ISSUE #19: Add the "Feeder" motor to the shooter subsystem
We are now running two shooter motors on the same side, the current implementation is good, because both shooter motors need to be characterized with velocity. Now we need to add another velocity motor called the feeder which will reliably feed a game piece to the shooter wheels. The feeder motor will require basic duty cycle input as well.

### PR DESCRIPTION
**Pull request by coolinm**
Feeder motor + feeder commands added

Tested and working in shop

### CODE REVIEW

File: src/main/java/frc/robot/commands/shooter/SetFeederMotorPercent.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import frc.robot.constants.shooter.ShooterConstants.MOTOR;
+import frc.robot.subsystems.Shooter;
+
+public class SetFeederMotorPercent extends Command
+{
+  Shooter m_Shooter;
+  double m_Percent;
+
+  /**
+   * @brief Applies a percent of the available voltage to shooter motor 1.
```
Comment (jasonsoohoo) [inline src/main/java/frc/robot/commands/shooter/SetFeederMotorPercent.java]: You haven't updated your documentation.

File: src/main/java/frc/robot/commands/shooter/SetFeederMotorVelocity.java [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+import frc.robot.constants.shooter.ShooterConstants.MOTOR;
+import frc.robot.subsystems.Shooter;
+
+public class SetFeederMotorVelocity extends Command
+{
+  Shooter m_Shooter;
+  double m_Velocity;
+
+  /**
+   * @brief Runs motor number 1 at a given velocity.
```
Comment (jasonsoohoo) [inline src/main/java/frc/robot/commands/shooter/SetFeederMotorVelocity.java]: Update your documentation

**Review by jasonsoohoo — CHANGES_REQUESTED** [review]
Requires changes.

**Comment by coolinm** [comment]
Changes made.

**Review by jasonsoohoo — APPROVED** [review]
Code is approved. Please test before merging.

**Review by Ferris-Bueller7 — APPROVED** [review]

**Pull request merged**